---
title: "GWAS Problem Set"
output: html_document
---
###R Markdown Tips

This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.

When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:

```{r}
summary(cars)
```

You can also embed plots, for example:

```{r, fig.width=4, fig.height=4, echo=FALSE}
plot(cars)
```

Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.

###Problem Set

All required data files are located in the GWAS_workshop.zip folder available at <https://github.com/hwheeler01/GWAS_workshop/blob/master/GWAS_workshop.zip>

CB1908 is a new chemotherapeutic agent developed by CubbyBear PharmaCorp.  Perform a GWAS on the IC<sub>50</sub> of CB1908-treated lymphoblastoid cell lines from the same Yoruba population used in the GWAS workshop (`genotype.*` files).  The phenotypes are located in `CB1908_IC50.txt` Answer the following questions about your analysis and embed relevant code and plots using R Markdown:


1.  Is the CB1908 IC<sub>50</sub> phenotype normally distributed?  Did you adjust the phenotype prior to running your analysis?

hist(data$CB1908_IC50,cex.lab=0.7,cex.main=0.7,cex.axis=0.7)

![Rplot01](https://user-images.githubusercontent.com/87441009/125724075-4a26f96a-499c-4e91-9e5e-16879ddf67b7.png)


Not normally distributed. Adjusting phenotype.

log2_CB1908_IC50 <- log2(data$CB1908_IC50)
hist(log2_CB1908_IC50,cex.lab=0.7,cex.main=0.7,cex.axis=0.7)

![Rplot](https://user-images.githubusercontent.com/87441009/125724091-f2a2c60d-578d-4706-bd93-163ac88715bb.png)


shapiro.test(data$CB1908_IC50)
w = 0.96818,p-value = 0.0005145
shapiro.test(log2_CB1908_IC50)
w = 0.99134, p-value = 0.3782 

P- value > 0.05 indicating normal distribution.


2.	Show the plink command you used to run your GWAS below and explain what each option within the command does.

plink --bfile genotypes --maf 0.05 --linear --adjust --pheno CB1908_IC50_log2.txt --pheno-name log2_CB1908_IC50 --out CB1908_L2_IC50

As in the GWAS_workshop

--bfile tells plink your data files that begin with genotypes are in binary format

--maf 0.05 tells plink to filter out any SNPs with a minor allele frequency < 0.05

--linear tells plink to run a linear additive association test for each SNP

--adjust tells plink to adjust the p-values for each SNP using various multiple-testing correction methods

--pheno tells plink the phenotype is located in a separate phenotype file

--pheno-name tells plink the column heading of the phenotype to use in the phenotype file (log2_CB1908_IC50)

--out tells plink the text to begin each output file (CB1908_L2_IC50)


3.	Make a Q-Q plot and a Manhattan plot of your results.  Embed the plots into this document and write a figure legend that explains each plot.

png(filename="qqplot1.png",res=100)
qq(data$P)
dev.off()

![qqplot1](https://user-images.githubusercontent.com/87441009/125723964-c94086da-99ce-4111-bc1e-9b83bbc8ba06.png)


png(filename="manplot1.png",res=100)
manhattan(data)
dev.off()

![manplot1](https://user-images.githubusercontent.com/87441009/125723987-5b104fbf-ae7a-477c-a9ab-4c46d0f6f24c.png)


4.	What does the Q-Q plot tell you about your data?  What potential confounders in your data should you analyze further? (Note: It is not necessary to actually perform the confounder analyses, just explain what you should further investigate.)

The Q-Q plot tells me my data is not normal. It is skewed to the right. Data on the right of the mode deviates from normal distribution. Confounders may be located to the right of the mode in the data.

5.	Do you have any genome-wide significant hits?  How did you correct for multiple hypothesis testing?  

Looking at the Manhattan plot, chromosomes 12 and 18 seem to indicate genome-wide significant hits. Adjusting the data in the beginning and checking the p-value corrected for multiple hypothesis testing.

6.	Is there any functional information known about your top SNPs?  Are they located in or near a gene?  How might these loci be functioning in response to CB1908?  Please use screen captures of appropriate databases or LocusZoom to highlight interesting loci.

7.	Make a plot and figure legend of your top hitâ€™s genotype vs. phenotype.  Is the minor allele associated with resistance or sensitivity to Drug X?

8.	What analyses and experiments would you do next to follow-up on your findings?


